frontend/components/AwardPdf.js

```diff
@@ -2,6 +2,59 @@ import React from 'react';
 import moment from 'moment';
 import { PDFDownloadLink, Page, Text, View, Document, StyleSheet, Font} from '@react-pdf/renderer';
 
+// const styles = StyleSheet.create({
```

Comment:
just to save and work off a copy for now?

Reply:
Yes, exactly.